xml/ha_config_basics.xml

```diff
@@ -698,6 +694,11 @@
    <sect3 xml:id="sec.ha.config.basics.resources.advanced.masters">
     <title>Multi-state Resources</title>
     <para>
+    <remark>taroth 2018-03-09: ygao, I guess this needs to be updated.
+    Could you suggest a new text, please? Also, the 'Pacemaker
+    Explained' Guide from upstream has been deprecated. Is there a new
+    place where one can find documentation on promotable clones (aka multi-state
+    resources) upstream?</remark>
```

Comment:
The changes of upstream docs are in here:
https://github.com/ClusterLabs/pacemaker/commit/8ab094345

, especially in doc/Pacemaker_Explained/en-US/Ch-Advanced-Resources.txt.

Reply:
Thanks. Also found out that the document 'Pacemaker Explained' still exists (at least for PM 1.1, an update for PM 2.0 has not been published yet). It now lives at http://clusterlabs.org/pacemaker/doc/ - updated the link and text accordingly.